Plugins/FpeMonitoring/CMakeLists.txt

```diff
@@ -0,0 +1,106 @@
+add_library(
+	ActsPluginFpeMonitoring SHARED
+	src/FpeMonitor.cpp)
+target_include_directories(
+  ActsPluginFpeMonitoring
+  PUBLIC
+    $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include/>
+    $<INSTALL_INTERFACE:include>)
+
+target_link_libraries(
+  ActsPluginFpeMonitoring PUBLIC ActsCore)
+
+# Fpe flags
+set(_fpe_options "")
+
+find_library(dl_LIBRARY dl)
+find_package(Backtrace)
+find_program(addr2line_EXECUTABLE addr2line)
+if(APPLE)
+  list(APPEND _fpe_options -D_GNU_SOURCE)
+else()
+
+  if(dl_LIBRARY)
```

Comment:
else if?

Reply:
Somehow in hierarchy cases like this I like a separate if. But if you prefer, I'll change it.